server.js

```diff
@@ -4960,6 +4960,27 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Facebook integration.
+camp.route(/^\/facebook\/share\/([^\/]+)\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var scheme = match[1]; // eg, https
+  var path = match[2];   // eg, shields.io
+  var format = match[3];
+  var page = encodeURIComponent(scheme + '://' + path);
+  var badgeData = getBadgeData('Share', data);
+  if (badgeData.template === 'social') {
+    badgeData.logo = badgeData.logo || logos.facebook;
+    badgeData.links = [
+      'https://www.facebook.com/sharer/sharer.php?u=' + page,
+      'https://www.facebook.com/search?q=' + page,
+     ];
+  }
+  badgeData.text[1] = '';
```

Comment:
Yes, there is: https://graph.facebook.com/?id=http://shields.io. 

```json
{
   "og_object": {
      "id": "608857025827761",
      "description": "We serve fast and scalable informational images as badges for GitHub, Travis CI, Jenkins, WordPress and many more services. Use them to track the state of your projects, or for promotional purposes.",
      "title": "Shields.io: Quality metadata badges for open source projects",
      "type": "website",
      "updated_time": "2017-01-10T03:04:55+0000"
   },
   "share": {
      "comment_count": 0,
      "share_count": 122
   },
   "id": "http://shields.io"
}
```
This could be used for a new share-with-count badge.